src/controllers/requestController.js

```diff
@@ -0,0 +1,26 @@
+import RequestService from '../services/requestService';
+import Response from '../utils/response';
+
+const { success, badRequest } = Response;
+
+/**
+ * Trip requests
+ */
+export default class RequestController {
+/**
+ * @param {object} body request body
+ * @param {object} res response body
+ * @returns {object} res
+ */
+  static async oneWay({ body }, res) {
+    try {
+      const data = await RequestService.oneway(body);
+
+      success(res, data, 201);
+    } catch ({ message: error }) {
+      if (error === 'You\'ve already booked this trip') return badRequest(res, error, 409);
```

Comment:
Why are you asserting if the error === `...`, I have given you this feedback before. Please throw an error in the services and catch it here.

Reply:
I'm using that assertion to throw a 409 for that particular error, I'll find a way around it

Reply:
return a 409